installutils/helmchart/manifests_test.go

```diff
@@ -91,4 +91,30 @@ metadata:
 
 		})
 	})
+
+	Context("load from github", func() {
+		It("works for a single chart", func() {
+			ref := GithubChartRef{
+				Ref:            "c00d740c83ec5b5439bf2f58fee110a546af7177",
```

Comment:
good idea locking to a ref, was going to suggest you make some dummy helm resources in this repo's test resources directory. Might still be worth doing so you can test various helm ignore rules.
Though it produces a chicken-and-egg problem, can't test against it till it merges

Reply:
FWIW the helm ignore rules are designed to be used during packaging, not rendering. So I'm not sure there's a ton of value in adding testing around that. That said, I did take a look at a few of the charts and rendered manifests just to make sure things checked out.